### ISSUE TITLE
[bug] cannot rearrange notes in 44.6 on Windows desktop

### ISSUE THREAD

**Issue by elephant2000** [zadam/trilium#1296]
It is no longer possible to rearrange notes by dragging and dropping them in the note tree. I recently updated from 43.4 to 44.6 on my Windows desktop. I tried to downgrade to 44.4, but it had the same thing…

By the way, Trilium is amazing, I use it all the time. ♥

**Comment by zadam** [comment]
Hi, could you post some details on this bug (or ideally a video)? Since to me it seems like drag & drop is working fine ...

**Comment by elephant2000** [comment]
Hi zadam, yes, after having downgraded back to 43.4, I upgraded again to 44.6 just now, and it could be relevant to mention that the following error message was prompted when first starting 44.6.

![image](https://user-images.githubusercontent.com/72633494/95666899-d6974000-0b5e-11eb-8459-5621a1610fc6.png)

I recorded a video of the behavior. After playing around with it, I can observe that the bug has some dynamic to it. A pattern that we can see in the video is that it is possible to move a note above or below a given note if the given note has, or have had, a child note.

Also worth to mention – it is the same in unhoisted mode. I hoisted the note for privacy reasons only.

I am very keen to provide more details if that is needed. I will be on a tight schedule in the following days, but I will get back to you asap if you have more requests.

https://youtu.be/Dsc_tfM-12A
